contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -428,11 +428,16 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
     function _startAuction() private {
```

Comment:
We need to make this change in the other vault contracts too.
`STETHVault`, `TreasuryVault`, `YearnVaults`